elide-core/src/main/java/com/yahoo/elide/security/executors/ActivePermissionExecutor.java

```diff
@@ -418,10 +421,12 @@ private ExpressionResult executeExpressions(final Expression expression,
                 result = expression.evaluate(Expression.EvaluationMode.ALL_CHECKS);
                 if (result == FAIL) {
                     ForbiddenAccessException e = new ForbiddenAccessException(
-                        annotationClass.getSimpleName(),
+                            getSimpleName(annotationClass),
```

Comment:
I'd rather not use a static import here, the fully qualified method is more clear.